app/src/main/java/com/locoquest/app/ui/theme/Theme.kt

```diff
@@ -28,7 +28,7 @@ private val LightColorPalette = lightColors(
 )
 
 @Composable
-fun IndividualAssignment304832Theme(
+fun locoQuestTheme(
```

Comment:
Think this was auto-generated as boilerplate code, but I updated it so the method is the app name + "theme" and followed the camel case convention!

https://kotlinlang.org/docs/coding-conventions.html#choose-good-names

Reply:
I agree that this looks better, but composable functions should be UpperCamelCase.  Thus, it should be LocoQuestTheme.